src/test/java/org/llorllale/cactoos/matchers/AssertionTest.java

```diff
@@ -44,7 +41,6 @@
  */
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
```

Comment:
It looks like JavadocMethodCheck can be removed as well here

Reply:
@ilyakharlamov fixed